mail-storage/src/main/java/mail/storage/exception/MessageExceptionInfo.java

```diff
@@ -0,0 +1,17 @@
+package mail.storage.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class MessageExceptionInfo {
+    private LocalDateTime timeStamp;
```

Comment:
these fields can be final - they are mandatory

Reply:
Fixed.